Lesson1/MyPlayground.playground/Contents.swift

```diff
@@ -1,2 +1,73 @@
-import UIKit
+var inputData: [Int] = [1, 2, 4, 1]
 
+/// The mean for an array of integers
+/// - Parameter inputData:The array of integers with an amount greater than 2
+/// - Returns: The mean; otherwise, nil.
+func mean(for inputData: [Int]) -> Double? {
+    if inputData.count < 2 { return nil }
```

Comment:
I see that doc comment says: `inputData:The array of integers with  an amount greater than 2`.

But for me it looks like a hack :)
Why you return `nil` for an array with `count == 1`?
What is the mean of the sequence `[1]`? It's not `nil`, it's `1`, right?

Please update the code to handle one-element arrays.

Reply:
Here and below.

Reply:
fixed